apps/cowswap-frontend/src/modules/tradeQuote/hooks/useQuoteParams.ts

```diff
@@ -1,35 +1,35 @@
 import { useMemo } from 'react'
 
-import { getAddress, getCurrencyAddress } from '@cowprotocol/common-utils'
+import { getCurrencyAddress, getWrappedToken } from '@cowprotocol/common-utils'
 import { PriceQuality } from '@cowprotocol/cow-sdk'
 import { useWalletInfo } from '@cowprotocol/wallet'
 
 import ms from 'ms.macro'
 
-import { LegacyFeeQuoteParams } from 'legacy/state/price/types'
-
 import { useAppData } from 'modules/appData'
 import { useDerivedTradeState } from 'modules/trade/hooks/useDerivedTradeState'
 
+import { FeeQuoteParams } from '../types'
+
 const DEFAULT_QUOTE_TTL = ms`30m` / 1000
 
-export function useQuoteParams(amount: string | null): LegacyFeeQuoteParams | undefined {
+export function useQuoteParams(amount: string | null): FeeQuoteParams | undefined {
   const { chainId, account } = useWalletInfo()
   const appData = useAppData()
 
   const state = useDerivedTradeState()
 
   const { inputCurrency, outputCurrency, orderKind } = state || {}
 
-  const sellToken = getAddress(inputCurrency)
+  const sellToken = inputCurrency ? getWrappedToken(inputCurrency).address : undefined
```

Comment:
This will make us quote using WETH address instead of the 0xeeeeeeee... address?
Probably backend does the same thing, but I its possible they take it into account for the estimation, right?

Reply:
<img width="1727" alt="image" src="https://github.com/user-attachments/assets/855eeab2-73a0-429d-8383-34d2453eb049" />

This is how it works and always has been (see the pic from prod). If you put 0xeeee as a token, the API will throw an error